webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/SegmentIndexFileService.scala

```diff
@@ -106,8 +106,10 @@ class SegmentIndexFileService @Inject()(config: DataStoreConfig,
         _ <- Fox.successful(())
         topLeftStart = buckets(bucketLocalOffset)(1)
         topLeftEnd = buckets(bucketLocalOffset)(2)
+        _ <- Fox
+          .bool2Fox(segmentIndex.reader.string().getAttr("/", "dtype_bucket_entries") == "uint16") ?~> "value for dtype_bucket_entries in segment index file is not supported, only uint16 is supported"
```

Comment:
```suggestion
       fileDtype <- tryo(segmentIndex.reader.string().getAttr("/", "dtype_bucket_entries"))
        _ <- Fox
          .bool2Fox(fileDtype == "uint16") ?~> f"value $fileDtype for dtype_bucket_entries in segment index file is not supported, only uint16 is supported"
```
(maybe something like this?)